src/window.rs

```diff
@@ -61,8 +61,10 @@ pub struct Window {
     font: Option<orbfont::Font>,
     pub widgets: RefCell<Vec<Arc<Widget>>>,
     pub widget_focus: Cell<usize>,
-    pub bg: Color,
-    pub running: Cell<bool>
+    pub bg: Cell<Color>,
+    pub running: Cell<bool>,
+    events: Vec<Event>,
```

Comment:
Ideally I'd like to get rid of mutability and internal mutability entirely, by moving towards what I describe in the PR description.